sound/soc/msm/msm8x16.c

```diff
@@ -51,13 +51,12 @@
 #define DEFAULT_MCLK_RATE 9600000
 
 #define EXT_CLASS_D_EN_DELAY 13000
-#define EXT_CLASS_D_DIS_DELAY 3000
```

Comment:
I had problems with delays in 3000. The code was tested before posting, so the delay and case 3 are needed.